robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -42,6 +49,20 @@ def set_enable_course_connections(self):
                 return
         raise Exception("Clicking element 'Enable Course Connections' failed after multiple tries")
 
+    def select_backfill_checkbox(self):
+        """ Selects the backfill checkbox """
+        locator = eda_lex_locators["eda_settings_cc"]["backfill_checkbox"]
+        self.selenium.wait_until_page_contains_element(locator)
+
+        for i in range(3):
+            i += 1
```

Comment:
You don't have to increment, the for loop will take care of it

Reply:
That makes the iterator variable unused (other than we can use it for logging the iteration). We could use while for retries but again it will go undefined limits of attempt until the condition passes

Reply:
Removed the iterator increment and replaced with logging